### ISSUE TITLE
Byte-order mark followed by space or tab results in Folia error

### ISSUE THREAD

**Issue by marijnschraagen** [LanguageMachines/ucto#79]
If a text file starts with a byte-order mark directly followed by a space or a tab then Folia gives an error. Column-based output is not affected. See the attached file as an example:
[bom.txt](https://github.com/LanguageMachines/frog/files/5343385/bom.txt)
This file starts with a byte order mark `EF BB BF`, then a space `20`, then `Jan loopt.`:
```
$ xxd bom.txt
efbb bf20 4a61 6e20 6c6f 6f70 742e 0a0a  ... Jan loopt...
```
This results in the following error:
```
$ frog -t bom.txt -X bom.xml
[...]
frog-:Wed Oct  7 21:34:35 2020 Frogging bom.txt
1			[﻿]	SPEC(symb)	1.000000	O	B-NP	0	ROOT
2	Jan	Jan	[Jan]	SPEC(deeleigen)	1.000000	B-PER	B-NP	3	su
3	loopt	lopen	[loop][t]	WW(pv,tgw,met-t)	0.998612	O	B-VP	0	ROOT
4	.	.	[.]	LET()	1.000000	O	O	3	punct

terminate called after throwing an instance of 'folia::ValueError'
  what():  TextContent: 'value' attribute may not be empty.
Afgebroken (geheugendump gemaakt)
```
Either removing the BOM or removing the space at the start (but leaving the BOM) results in succesful parsing of the file. A tab character instead of a space also triggers the issue.

**Comment by kosloot** [comment]
Interestingly, this bug is hard to reproduce in Ucto itself, as Ucto Frog uses the Ucto API slightly different from Ucto itself.
Still it is an Ucto problem and I will close this issue and we continue in https://github.com/LanguageMachines/ucto/issues/79

**Comment by kosloot** [comment]
I committed a fix in Ucto. This should solve the problem. Please test.

**Comment by marijnschraagen** [comment]
I tried to update lamachine but I got an error about installing Aptitude (even though I installed from scratch yesterday and everything worked smoothly). Maybe I can try updating just ucto using `lamachine-update --only`, which packages should I specify to get the new ucto only?

**Comment by kosloot** [comment]
@marijnschraagen  you have to wait until @proycon updates LaMachine. In this case the Development version, until the bug fix is approved and officially relased.
Hope @proycon reacts soon....

**Comment by proycon** [comment]
Sorry for the delay! Thanks for the fix @kosloot! I'm testing it right now and will do a release straight away if this indeed fixes it.

> I tried to update lamachine but I got an error about installing Aptitude (even though I installed from scratch yesterday and everything worked smoothly). 

That is strange, can you create an issue if it persists?

> Maybe I can try updating just ucto using lamachine-update --only, which packages should I specify to get the new ucto only?

You can do ``lamachine-update --only languagemachines-basic``, which includes frog and ucto. But it will only work if you're on the development version. Or just hold on until I publish the release and then it'll work in the stable LaMachine too.

**Comment by proycon** [comment]
The fix works and I have now released ucto v0.22, it should be available in LaMachine after a lamachine-update (or a fresh installation).
